packages/strapi-plugin-upload/admin/src/components/InputMedia/EmptyInputMedia.js

```diff
@@ -10,10 +15,46 @@ const EmptyInputMedia = styled.div`
   justify-content: center;
   align-items: center;
   border-radius: ${({ theme }) => theme.main.sizes.borderRadius};
-  background-color: ${({ theme }) => theme.main.colors.black};
+  background-color: ${({ theme, isInDropZone }) =>
+    isInDropZone ? '#515764' : theme.main.colors.black};
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
+
+  & * {
+    pointer-events: none;
+  }
 `;
 
+const EmptyInputMedia = ({ children, ...props }) => {
+  const parentRef = useRef(null);
```

Comment:
Why do you need a ref ?

Reply:
I don't 😛 . I was experimenting with the JS APIs and forgot to remove that one haha